modules/command-engine/core/src/main/scala/surge/core/event/SurgeEventServiceImpl.scala

```diff
@@ -0,0 +1,10 @@
+// Copyright © 2017-2021 UKG Inc. <https://www.ukg.com>
+
+package surge.core.event
+
+import akka.actor.ActorSystem
+import com.typesafe.config.Config
+import surge.internal.domain.SurgeMessagePipeline
+
+abstract class SurgeEventServiceImpl[Agg, Event](actorSystem: ActorSystem, businessLogic: SurgeEventServiceModel[Agg, Event], config: Config)
```

Comment:
Since this is an implementation would it make sense to be in one of the internal packages/package private to Surge?  Probably same with `SurgeCommandImpl` now that I'm looking at it.